pages/order.tsx

```diff
@@ -642,6 +635,26 @@ const Landing: NextPage = () => {
   );
 };
 
+const StyledDiv = styled.div`
+  ${Mixins.flex('row')}
+`; 
+
+const StyledDropdown = styled(Dropdown)<{ textMarginSize: string }>`
+  ${({ textMarginSize }): string => `
+    left: 1px;
+    margin: 0 ${textMarginSize} 0 ${textMarginSize};
+  `}
+`;
+
+const StyledSmallText = styled(SmallText)<{ isFromBot: boolean}>`
+  ${({ isFromBot }): string => `
+    text-align: ${isFromBot ? 'left' : 'right'};
+    color: ${isFromBot ? 'black' : 'white'};
+  `}
+  white-space: pre-line;
+  margin: 0 2px 0 2px;
+`
```

Comment:
If there is styled components in your application it means you can convert to components in storybook. I'll talk about this today for our lesson